js/start.js

```diff
@@ -18,11 +18,13 @@ import Followers from './collections/Followers';
 import listingDeleteHandler from './startup/listingDelete';
 import { fetchExchangeRates } from './utils/currency';
 import './utils/exchangeRateSyncer';
-import './utils/listingData';
+import { getBody } from './utils/listingData';
```

Comment:
getBody() is in utils/selectors.js. Did this code work when you ran it, or are you cowboy dev'in? lol